lib/live_ex_webrtc/publisher.ex

```diff
@@ -231,8 +242,8 @@ defmodule LiveExWebRTC.Publisher do
       pc_genserver_opts: Keyword.get(opts, :pc_genserver_opts, [])
     }
 
-    # Check the "Record stream?" checkbox by default if recorder was configured
-    record? = publisher.recorder != nil
+    # Check the "Record stream?" checkbox by default if recordings are allowed 
+    record? = publisher.recordings? != nil
```

Comment:
It's a boolean, right?
```suggestion
    # Check the "Record stream?" checkbox by default if recordings are allowed 
    record? = publisher.recordings? == true
```